analytics_data_api/v0/views/courses.py

```diff
@@ -689,6 +689,54 @@ def get_queryset(self):
         return rows
 
 
+# pylint: disable=abstract-method
+class ProblemsAndTagsListView(BaseCourseView):
+    """
+    Get the problems with the connected tags.
+
+    **Example request**
+
+        GET /api/v0/courses/{course_id}/problems_and_tags/
+
+    **Response Values**
+
+        Returns a collection of submission counts and tags for each problem. Each collection contains:
+
+            * module_id: The ID of the problem.
+            * tag_name: Name of the connected tag
+            * tag_value: Value of the connected tag
+            * total_submissions: Total number of submissions
+            * correct_submissions: Total number of *correct* submissions.
+    """
+    serializer_class = serializers.ProblemsAndTagsSerializer
+    allow_empty = False
+    model = models.ProblemsAndTags
+
+    def get_queryset(self):
+        queryset = self.model.objects.filter(course_id=self.course_id)
+        items = queryset.all()
+
+        result = {}
+
+        for v in items:
+            if v.module_id not in result:
```

Comment:
Nit: I find it a bit clearer if the `if` statements can be made in the positive.  My suggestion would be to change this to `if v.module_id in result:` likewise move the code blocks.  What do you think?